packages/node_modules/@webex/webex-core/test/unit/spec/services/services.js

```diff
@@ -100,6 +101,34 @@ describe('webex-core', () => {
       });
     });
 
+    describe('#fetchLoginUserInfomation', () => {
+      it('Fetch login users information ', async () => {
+        webex.request = sinon.stub().returns(Promise.resolve({body: {userPreferences: 'userPreferences'}}));
+
+        const res = await services.fetchLoginUserInfomation();
+
+        assert.calledWith(webex.request, {
+          method: 'GET',
+          service: 'identity',
+          resource: 'identity/scim/v1/Users/me?showAllTypes=true'
+        });
+        assert.isDefined(res);
```

Comment:
we could check if res matches what it should be `{userPreferences: 'userPreferences'}`